.wiki/setting-up-the-stripe-gateway-v2.md

```
@@ -36,21 +36,21 @@ You can either use **Test Data** (to test if the integration is working on your
 
 ![](https://wp-ultimo-space.fra1.cdn.digitaloceanspaces.com/hs-file-kdVC3W8Bsr.png)
 
-Copy the value from the **Publishable key** and **Secret key** , from the **Token** column and paste it on WP Multisite WaaS Stripe Gateway fields. Then click to **Save Changes**.
+Copy the value from the **Publishable key** and **Secret key** , from the **Token** column and paste it on Multisite Ultimate Stripe Gateway fields. Then click to **Save Changes**.
 
 ![](https://wp-ultimo-space.fra1.cdn.digitaloceanspaces.com/hs-file-JyAifSGNOn.png)
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)